<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeriesAttributes</class>
 <widget class="QWidget" name="SeriesAttributes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="MediaControl" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>34</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>34</width>
         <height>34</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background: white;</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/create/area chart</normaloff>:/create/area chart</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="TitleLabel" name="label">
       <property name="class" stdset="0">
        <string>heading</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>376</width>
            <height>191</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Basic properties</string>
             </property>
             <property name="class" stdset="0">
              <string>section</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <property name="labelAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <property name="verticalSpacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="Label" name="label_3">
               <property name="text">
                <string>Name</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="Label" name="label_4">
               <property name="text">
                <string>Visible in editor</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="Label" name="label_5">
               <property name="text">
                <string>Visible in renderer</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="LineEdit" name="lineEdit"/>
             </item>
             <item row="1" column="1">
              <widget class="ComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>Default</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>On</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Off</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="ComboBox" name="comboBox_2">
               <item>
                <property name="text">
                 <string>Default</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>On</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Off</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Object</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea_3">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>376</width>
            <height>191</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_41">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Object properties</string>
             </property>
             <property name="class" stdset="0">
              <string>section</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout_3">
             <property name="labelAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="formAlignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="horizontalSpacing">
              <number>6</number>
             </property>
             <property name="verticalSpacing">
              <number>6</number>
             </property>
             <item row="0" column="0">
              <widget class="Label" name="label_18">
               <property name="text">
                <string>Color</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="Label" name="label_12">
               <property name="text">
                <string>Opacity</string>
               </property>
               <property name="class" stdset="0">
                <string>item</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="Picker" name="comboBox_3"/>
             </item>
             <item row="1" column="1">
              <widget class="DoubleSpinBox" name="doubleSpinBox">
               <property name="suffix">
                <string>%</string>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>100.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>99</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MediaControl</class>
   <extends>QPushButton</extends>
   <header>mediacontrol.h</header>
  </customwidget>
  <customwidget>
   <class>Label</class>
   <extends>QLabel</extends>
   <header>label.h</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>DoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>doublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>titlelabel.h</header>
  </customwidget>
  <customwidget>
   <class>Picker</class>
   <extends>QComboBox</extends>
   <header>picker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
